Implement_A_Stack/myDataStructs.py

```diff
@@ -1,102 +1,94 @@
 # -*- coding: utf-8 -*-
-"""
-Spyder Editor
-
-This is a temporary script file.
-"""
 import copy
 
-class myStack(object):
-    'My stack implementation'
+class MyStack(object):
+    """My stack implementation"""
+
     def __init__(self):
-        self.contents = ()
+        self.__contents = ()
 
 
     def empty(self):
-        if len(self.contents):
-            return False
-        else:
-            return True
+        return len(self.__contents) == 0
 
 
     def push(self, val):
-        self.contents = (val, self.contents)
+        self.__contents = (val, self.__contents)
 
 
     def pop(self):
         try:
-            val = self.contents[0]
-            self.contents = self.contents[1]
+            val = self.__contents[0]
+            self.__contents = self.__contents[1]
             return val
         except:
             return None
 
 
     def peek(self):
-        val = self.pop()
-        self.push(val)
-        return val
+        try:
+            return self.__contents[0]
+        except:
+            return None
```

Comment:
Popping and re-pushing adds unnecessary complexity, though I see where you were coming from for DRYness. Could break out the WET part as a decorator?